pkg/util/kubernetes/apiserver/apiserver.go

```diff
@@ -273,9 +276,19 @@ func (c *APIClient) GetInformerWithOptions(resyncPeriod *time.Duration, options
 	return informers.NewSharedInformerFactoryWithOptions(c.InformerCl, *resyncPeriod, options...)
 }
 
+func (c *APIClient) GetRestConfig() *rest.Config {
```

Comment:
Not sure you should expose `Config` rather have a method to return a SPDY client, same for RESTClient

Reply:
Let's chat offline, I'm not sure how I can extract it cleanly 🤔

Reply:
Done 👍🏻